src/selectors/expandThoughts.ts

```diff
@@ -143,9 +146,11 @@ function expandThoughtsRecursive(
         const isAncestor = () => isDescendant(childContext, expansionBasePathContext)
 
         /** Check if the path is equal to the expansion path. */
-        const isExpansionBasePath = () => equalArrays(childContext, expansionBasePathContext)
+        const isExpansionBasePath = equalArrays(childContext, expansionBasePathContext)
```

Comment:
You'll want to keep `isExpansionBasePath` a function so that `equalArrays` can be short-circuited when possible (due the order of the return condition. This is for performance reasons. `equalArrays` is O(n).

Reply:
I see. Making the change now.